generators/entity/index.js

```diff
@@ -83,20 +83,39 @@ module.exports = EntityGenerator.extend({
             defaults: ''
         });
 
+        // This adds support for a `--skip-client` flag
```

Comment:
I would prefer this to be read from yo-rc file rather than passing this for each entity as this should be aligned to the app structure itself